fMRIVolume/scripts/IntensityNormalization.sh

```diff
@@ -36,18 +36,23 @@ Usage() {
   echo "             [--inscout=<input name for scout image (pre-sat EPI)>]"
   echo "             [--oscout=<output name for normalized scout image>]"
   echo "             [--workingdir=<working dir>]"
+  echo "             [--boldmask=<what mask to use for generating final BOLD timeseries:"
+  echo "                         T1_fMRI_FOV (default): mask based on T1 and voxels available at all timepoints (i.e., the fMRI FOV)"
+  echo "                         T1_DILATED_fMRI_FOV: a once dilated T1w brain based mask combined with fMRI FOV"
+  echo "                         T1_DILATED2x_fMRI_FOV: a twice dilated T1w brain based mask combined with fMRI FOV,"
+  echo "                         fMRI_FOV: a fMRI FOV mask"
 }
 
 # function for parsing options
 getopt1() {
     sopt="$1"
     shift 1
     for fn in $@ ; do
-	if [ `echo $fn | grep -- "^${sopt}=" | wc -w` -gt 0 ] ; then
-	    echo $fn | sed "s/^${sopt}=//"
-	    # if [ ] ; then Usage ; echo " " ; echo "Error:: option ${sopt} requires an argument"; exit 1 ; end
-	    return 0
-	fi
+  if [ `echo $fn | grep -- "^${sopt}=" | wc -w` -gt 0 ] ; then
+      echo $fn | sed "s/^${sopt}=//"
+      # if [ ] ; then Usage ; echo " " ; echo "Error:: option ${sopt} requires an argument"; exit 1 ; end
+      return 0
+  fi
```

Comment:
The tab to space conversion messed up the indentation here.

I would also add quotes to line 50, making it `for fn in "$@" ; do`, which should allow it to handle quoted whitespace correctly.  No, I don't expect this change everywhere, but if you are editing it anyway...

Reply:
And, for completeness, add quotes to line 52, making it `echo "$fn" | sed "s/^${sopt}=//"`.  Line 51 will work as long as there is no whitespace in the flag itself (and if it were to be changed, it deserves a full rewrite to simplify).